portality/templates/publisher/preservation.html

```diff
@@ -0,0 +1,117 @@
+    <fieldset>
+      <h3 class="form__header">Preservation File</h3>
+      <div class="form__question">
+        <label for="upload-xml-file">Select a file</label>
+        <input type="file" id="upload-xml-file" name="file">
+        <p><small>Must be less than 50Mb.</small></p>
+      </div>
+    </fieldset>
+    <button type="submit" id="preservation" class="button button--primary">Upload</button>
+  </form>
+</div>
+
+<hr>
+
+{% if previous|length > 0 %}
+<h2>History of uploads (showing last {{previous|length}})</h2>
+
+<table id="previous_files" style="width: 100%; table-layout:fixed" class="tablesorter">
+    <thead>
+        <tr>
+            <th style="width: 17%">Upload Date</th>
+            <th style="width: 20%">Filename</th>
+            <th style="width: 15%">Upload Status</th>
+            <th style="width: 36%">Notes</th>
+        </tr>
+    </thead>
+{% for file in previous %}
+    <tr>
+        <td>{{ file.created_timestamp | utc_timestamp("%d %b %Y %H:%M:%S %Z") }}</td>
+        <td style="word-wrap: break-word">
+            {{file.filename}}
+        </td>
+        <td>
+            {% if file.status == "initiated" %}
+                <span style="color: #000066">initiated</span>
+            {% elif file.status == "validated" %}
+                <span style="color: #000066">pending</span>
+            {% elif file.status == "pending" %}
+                <span style="color: #000066">pending</span>
+            {% elif file.status == "failed" %}
+                <span style="color: #660000"><span data-feather="alert-octagon" aria-hidden="true"></span>processing failed</span>
+            {% elif file.status == "uploaded" %}
+                <span style="color: #006600"><span data-feather="thumbs-up" aria-hidden="true"></span>successfully processed</span>
+            {% endif %}
+        </td>
+        <td style="word-wrap:break-word">
+            {% if file.error %}
+                {{file.error}}
+            {% else %}
+                {% if file.status == "uploaded" %}
+                    uploaded to archive server
+                {% endif %}
+                {% if file.status == "initiated" or file.status == "pending" %}
+                    upload to archive server is in process
+                {% endif %}
+            {% endif %}
+            {%  if file.error_details %}
+                <div id="details_{{ file.id }}" style="display:none">
+                    {% if file.error_details %}
+                        {{ file.error_details }}
+                    {% endif %}
+                </div>
+                <a href="#" class="show_error_details" data-id="{{ file.id }}">(show error details)</a>
+            {% endif %}
+        </td>
+    </tr>
+{% endfor %}
+</table>
+{% endif %}
+
+{% endblock %}
+
+{% block extra_js_bottom %}
+<script type="text/javascript">
+$(document).ready(function() {
+
+document.getElementById("crossref-alert").style.display = "none";
```

Comment:
This isn't required - it's from the metadata upload page where it cares about whether the crossref xml schema is loaded or not